src/components/utils/hometabFilterEnum.js

```diff
@@ -0,0 +1,6 @@
+// ENUMS that represent the tab a user is on for home page navigation
+export const tabFilters = {
```

Comment:
History should be in all caps and also it could look like this:

JOB: "active",
HISTORY: "completed"

So then you don't have to use this conditional : 

![image](https://user-images.githubusercontent.com/59713270/220829055-30b81e36-fa73-4433-b8ad-3bfe041d084b.png)

Can just use `selectedTabFilter`